source/ncentSDK.js

```diff
@@ -3,7 +3,7 @@ const StellarSdk = require('stellar-sdk');
 const signObject = require('./utils').signObject;
 
 // const defaultNet = 'http://localhost:8010/api';
-const defaultNet = 'http://18.219.87.29:8010/api';
+const defaultNet = "http://18.223.237.17:8010/api";
```

Comment:
should this be in an env var?

Reply:
If we are mirroring the version of our SDK that will be published as a public node module, I don't believe there will be a way to prevent having this IP in some sort of file.

Reply:
Until we use API Gateway, we will be forced to push an update to the public node module any time the public IP changes (which will happen every time we update the docker image and run a new ECS task)

If you want, I can just start with the API Gateway stuff and we can come back to this once we have a static hostname; however, that will won't be stored in an ENV variable (unless there is a way to do that which I'm not thinking of)